libafl/src/stages/mod.rs

```diff
@@ -540,16 +540,27 @@ impl RetryRestartHelper {
     }
 }
 
+/// The index of a stage
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash, PartialOrd, Ord, Serialize, Deserialize)]
+#[repr(transparent)]
+pub struct StageIdx(pub(crate) usize);
```

Comment:
Since we have `CorpusId`, I think this one should be `StageId` as well

Reply:
I guess an id and an index are slightly different, but...
Otherwise StageIndex?